cantools/database/can/templates/DBC.h

```diff
@@ -124,10 +124,10 @@ class Frame {
   uint32_t cycle_time() const { return cycle_time_; }
 
   /** Getter to front of underlying data buffer. */
-  uint8_t* buffer() { return &buffer_[0]; }
+  uint8_t* buffer() const { return &buffer_[0]; }
```

Comment:
Will const qualifying this function break things if this function is used to access the underlying buffer and populate it? Or does it not matter because the buffer returned isn't const?

Reply:
Not sure if const qualifying does something special in that it means member variables of class not mutable but we are using this to populate `buffer_`

Reply:
I think const qualifying a method just means nothing is mutated within the method call? As long as the return type does not have a const specifier I think it's ok

Reply:
I can also drop this change for now. I added this as part of the work to send the A200 PGNs, but just smashed it in here to save putting up another PR. FWIW it compiled fine with these changes

Reply:
If unit tests passed it should be fine!